src/julienne/julienne_test_harness_s.F90

```diff
@@ -47,8 +47,12 @@
           print *
           print '(*(a,:,i0))', "_____ ", passes, " of ", tests, " tests passed. ", skips, " tests were skipped _____"
```

Comment:
Looking now I also see that the line above printed by image 1 may incorrectly report that all tests passed, even when one or more test failures occurred only on other images (a rather misleading output in that case). Longer-term it might make sense to either preserve the results of the earlier reductions, or perform a new reduction here to print the actual number of tests that passed on all images.